### DISCUSSION TITLE
Setup Owncloud infinite scale with docker compose

### DISCUSSION THREAD

**Discussion by giscus** [FarisZR/FarisZR.com#12] [Comments]
# Setup Owncloud infinite scale with docker compose

a quick guide on setting up ownCloud infinite scale using docker compose with additional services like Apache Taki for better full search and Libreoffice.

https://fariszr.com/owncloud-infinite-scale-docker-setup/

**Comment by wawefr** [comment]
Hello thanks for your explanations Good job. I such to activate webdav but I don't found any tuto. Do you test it ? Thanks (I'm a just a geek who mount a NAS )

**Reply by FarisZR** [reply]
Unfortunately I didn't go further into OIS due to the disappointing mobile app, you will have to look in the official documentation for this.

**Reply by wawefr** [reply]
thanks 
difficult to find tutoriels 
have a good day
